cardano-node/ChangeLog.md

```diff
@@ -15,7 +15,7 @@ None
 - New P2P topology file format. See [issue #4563][#4563] or the [config
```

Comment:
where is issue 4563?

Reply:
it should be issue #4559 (#4563 was a pr which addressed it).

Reply:
Fixed.